nav2_map_server/src/mapio.cpp

```diff
@@ -0,0 +1,482 @@
+/* Copyright (c) 2020 Samsung R&D Institute Russia
+ * Copyright 2019 Rover Robotics
+ * Copyright 2018 Brian Gerkey
```

Comment:
2018 Brian Gerkey? Since when did Brian work on navigation code in recent memory

Reply:
His name appeared in e57809b3233ec646c457e9764be3d01b4c467335 port to navigation2. In ROS1 navigation stack the `image_loader.cpp` (currently `occ_grid_loader.cpp`) file was added with Brian as an author in a very first commit: https://github.com/ros-planning/navigation/commit/1f335323a605b49b4108a845c55a7c1ba93a6f2e. Unfortunately, I can not find any clue for earlier history. But I think we can consider him as an author of original map loading logic.

Reply:
Change the year to 2010 for Brian. I don't have any idea why someone change it to 2018.

Reply:
Done: https://github.com/ros-planning/navigation2/commit/89ece9672dc847161607acc658b63fe2bafb90a7#diff-77a6f7ce84dde783aad18c29015f94f5R2 and https://github.com/ros-planning/navigation2/commit/89ece9672dc847161607acc658b63fe2bafb90a7#diff-a42e5d09c9ddddb03729320e0be84f12R17